uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss

```diff
@@ -28,14 +26,13 @@
 
     .chr_admin-dashboard_side-nav {
       overflow: hidden;
-
-      ul {
-        max-width: $chr-admin-dashboard-side-nav-width;
-      }
+      width: 15%;
```

Comment:
These styles are taken from `contact-summary` page. There also we have a similar layout. The summation of both the number is 99 as 1 is left for the gap between the 2 panels.
<img width="929" alt="dashboard___hr17-1" src="https://user-images.githubusercontent.com/5058867/27837452-f4d3b946-6101-11e7-86f4-e72842b75a78.png">